pkg/controller/middleware/operating_system.go

```diff
@@ -25,9 +25,11 @@ import (
 
 func AddOperatingSystemFromUserAgent() mux.MiddlewareFunc {
 	userAgents := map[string]database.OSType{
-		"darwin": database.OSTypeIOS,
-		"iphone": database.OSTypeIOS,
-		"dalvik": database.OSTypeAndroid,
+		"darwin":                 database.OSTypeIOS,
+		"iphone":                 database.OSTypeIOS,
+		"alamofire":              database.OSTypeIOS,
+		"dalvik":                 database.OSTypeAndroid,
+		"androiddownloadmanager": database.OSTypeAndroid,
```

Comment:
I would be surprised to see this UI making verify calls to the verification server - please confirm

Reply:
good point, i got the user agent sample from the CDN.  i'll check on the apiserver side and update

Reply:
I think it might be okhttp that we're looking for, but need to gather some more data or check with the states to confirm.  I'll close this for now pending confirmation